esp-hal/src/pcnt/mod.rs

```diff
@@ -34,7 +34,7 @@
 //!
 //! // setup a pulse couter
 //! println!("setup pulse counter unit 0");
-//! let pcnt = PCNT::new(peripherals.PCNT);
+//! let pcnt = PCNT::new(peripherals.PCNT, Some(PCNT));
```

Comment:
Okay, seems my first comment got lost :D 
Maybe we could be more consistent with names here and in relevant example, i.e `Some(interrupt_handler)`